sources/GameLobby.cpp

```diff
@@ -0,0 +1,41 @@
+//
+// Created by haduluoke on 10/10/19.
+//
+
+#include "GameLobby.h"
+
+int GameLobby::createRoom() {
+    int rNumber;
+    int from = static_cast<int>(std::pow(10, roomNumberLength - 1));
```

Comment:
This may not produce what you want due to truncation error.
See https://stackoverflow.com/questions/18155883/strange-behaviour-of-the-pow-function